### PR TITLE
Luk10114/85 add features feature service

### PR DESCRIPTION
**Pull request by LDuncAndroid**
Ready for PR:
https://devtopia.esri.com/runtime/common-samples/issues/85

Thanks.

### CODE REVIEW

File: java/add-features-feature-service/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Add features (Feature Service)
+
+Add new features to an online feature service.
+
+![Add Features (Feature Service)](add-features-feature-service.png)
+
+## Use case
+
+A `FeatureLayer` created using this `SeviceFeatutreTable` that is applied to the `ArcGISMap` will display any new Features automatically.
```
Comment (TADraeseke) [inline java/add-features-feature-service/README.md]: ServiceFeatureTable

File: java/add-features-feature-service/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## How it works
+
+1. Create a service feature table from a URL, `new ServiceFeatureTable("URL")`
```
Comment (TADraeseke) [inline java/add-features-feature-service/README.md]: Create a new `ServiceFeatureTable` from a URL.

File: java/add-features-feature-service/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+1. Create a `FeatureLayer` from the service feature table, `new FeatureLayer(ServiceFeatureTable)`
```
Comment (TADraeseke) [inline java/add-features-feature-service/README.md]: as above

File: java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+public class MainActivity extends AppCompatActivity {
+
+  private static final String TAG = MainActivity.class.getSimpleName();
+
+  private MapView mMapView;
+
+  private ServiceFeatureTable mServiceFeatureTable;
+
+  @SuppressLint("ClickableViewAccessibility")
```
Comment (TADraeseke) [inline java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java]: remove suppress lint

File: java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    // the service feature table
+    mMapView.setOnTouchListener(new DefaultMapViewOnTouchListener(this, mMapView) {
+      @Override public boolean onSingleTapConfirmed(MotionEvent event) {
+        // create a point from where the user clicked
+        android.graphics.Point point = new android.graphics.Point((int) event.getX(), (int) event.getY());
+
+        // create a map point from a point
+        Point mapPoint = mMapView.screenToLocation(point);
+
+        // for a wrapped around map, the point coordinates include the wrapped around value
```
Comment (TADraeseke) [inline java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java]: lets remove this for this sample too

File: java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java (hunk 4 above)
Comment (TADraeseke) [inline java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java]: Remove suppress lint

Comment (TADraeseke) [inline java/add-features-feature-service/src/main/java/com/esri/arcgisruntime/sample/addfeaturesfeatureservice/MainActivity.java]: remove suppress lint

File: kotlin/add-features-feature-service/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# Add features (Feature Service)
+
+Add new features to an online feature service.
+
+![Add Features (Feature Service)](add-features-feature-service.png)
+
+## Use case
+
+A `FeatureLayer` created using this `SeviceFeatutreTable` that is applied to the `ArcGISMap` will display any new Features automatically.
```
Comment (TADraeseke) [inline kotlin/add-features-feature-service/README.md]: as above

File: kotlin/add-features-feature-service/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+## How it works
+
+1. Create a service feature table from a URL, `ServiceFeatureTable("URL")`
```
Comment (TADraeseke) [inline kotlin/add-features-feature-service/README.md]: as above

**Comment by TADraeseke** [comment]
Just made a few small changes to READMEs. Good to merge. Thanks @LDuncAndroid!

**Comment by LDuncAndroid** [comment]
Thanks @TADraeseke

**Pull request merged**